packages/cli/src/api/formats/po-gettext.ts

```diff
@@ -0,0 +1,295 @@
+            key
+          )
+        }
+        item.msgstr = [message.translation]
+      }
+
+      item.comments = message.comments || []
+      item.extractedComments = message.comment ? [message.comment] : []
+      item.references = message.origin ? message.origin.map(joinOrigin) : []
+      // @ts-ignore: Figure out how to set this flag
+      item.obsolete = message.obsolete
+      item.flags = message.flags
+        ? R.fromPairs(message.flags.map((flag) => [flag, true]))
+        : {}
+      return item
+    })
+  )(items)
+
+const getMessageKey = R.prop<"msgid", string>("msgid")
+const getTranslations = R.prop<"msgstr", string[]>("msgstr")
+const getExtractedComments = R.prop("extractedComments")
+const getTranslatorComments = R.prop("comments")
+const getOrigins = R.prop("references")
+const getFlags = R.compose(
+  R.map(R.trim),
+  R.keys,
+  R.dissoc("obsolete"), // backward-compatibility, remove in 3.x
+  R.prop("flags")
+)
+const isObsolete = R.either(R.path(["flags", "obsolete"]), R.prop("obsolete"))
+
+const getTranslationCount = R.compose(R.length, getTranslations)
+
+const deserialize: (Object) => Object = R.map(
+  R.applySpec({
+    translation: R.compose(R.head, R.defaultTo([]), getTranslations),
+    comment: R.compose(R.head, R.defaultTo([]), getExtractedComments),
+    comments: (item) =>
+      R.concat(
+        getTranslatorComments(item) as string,
+        R.tail(getExtractedComments(item))
+      ),
+    obsolete: isObsolete,
+    origin: R.compose(R.map(splitOrigin), R.defaultTo([]), getOrigins),
+    flags: getFlags,
+  })
+)
+
+const convertPluralsToICU = (items: POItemType[], lang?: string) => {
+  // .po plurals are numbered 0-N and need to be mapped to ICU plural classes ("one", "few", "many"...). Different
+  // languages can have different plural classes (some start with "zero", some with "one"), so read that data from CLDR.
+  let pluralForms: string[] | null = []
+  if (lang) {
+    pluralForms = pluralsCldr.forms(lang)
+    // Plurals may be `null` if lang is not found. As long as no plural is used, don't report an error.
+  }
+
+  items.forEach((item) => {
+    const translationCount = getTranslationCount(item)
+    const messageKey = getMessageKey(item)
+
+    // Messages without multiple translations (= plural cases) need no further processing.
+    if (translationCount <= 1) {
+      return
+    }
+
+    // msgid_plural must be set, but its actual value is not important.
+    if (!item.msgid_plural) {
+      console.warn(
+        `Multiple translations for item with key "%s" but missing 'msgid_plural' is not supported and it will be ignored.`,
+        messageKey
+      )
+      return
+    }
+
+    const ctx = new URLSearchParams(item.msgctxt)
+
+    // If an original ICU was stored, use that as `msgid` to match the catalog that was originally exported.
+    const storedICU = ctx.get("icu")
+    if (storedICU != null) {
```

Comment:
!== ?

Reply:
As above, I believe this is valid use of `!=`

Reply:
@tricoder42 Yep, that's the (only) way I use `==` for, as is most cases both `null` and `undefined` have the same meaning and the exact differentiation doesn't matter.